### DISCUSSION TITLE
Cannot install next/jest: npm error code 128

### DISCUSSION THREAD

**Discussion by rvk220** [vercel/next.js#68361] [Help] [answered]
### Summary

**Actual situation**: when I execute `npm i -D next/jest`, I have the following error:
```
npm error code 128
npm error An unknown git error occurred
npm error command git --no-replace-objects ls-remote ssh://git@github.com/next/jest.git
npm error git@github.com: Permission denied (publickey).
npm error fatal: Could not read from remote repository.
npm error
npm error Please make sure you have the correct access rights
npm error and the repository exists.
```
**Expected:** installation should be successful (as for any other module).

### Additional information

```js
What I tried (didn't help):
- updating NPM (current version: 10.8.2);
- cleaning NPM cache with `npm cache clean --force`.
```

**Comment by icyJoseph** [comment, marked as answer]
`next/jest` is not a package, it is a namespace within `next`, so if you have `next` installed, you have `next/jest` available:

- https://github.com/vercel/next.js/blob/canary/packages/next/src/build/jest/jest.ts
- https://github.com/vercel/next.js/blob/canary/packages/next/jest.js
